sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue

```diff
@@ -79,11 +100,38 @@ export default defineComponent({
     const snackBar = useSnackBar();
     const programYearOptions = ref([] as SelectItemType[]);
     const programYearId = ref();
+    const offeringIntensity = ref();
     const startApplicationForm = ref({} as VForm);
     const { checkNullOrEmptyRule } = useRules();
     const draftApplicationModal = ref({} as ModalDialog<boolean>);
+    const allowFulltime = ref(false);
```

Comment:
`allowFulltime` does not need to be reactive.
Besides that, the logic inside `watchEffect` can be moved to `onMounted` since it's not expected to change after the component is mounted.